src/js/setup-text-tracks.js

```diff
@@ -25,15 +25,42 @@ function attachDashTextTracksToVideojs(player, tech, tracks) {
   const tracksAttached = tracks
     // Map input data to match HTMLTrackElement spec
     // https://developer.mozilla.org/en-US/docs/Web/API/HTMLTrackElement
-    .map((track) => ({
-      dashTrack: track,
-      trackConfig: {
-        label: track.lang,
-        language: track.lang,
-        srclang: track.lang,
-        kind: track.kind
+    .map((track) => {
+      let localizedLabel;
+
+      if (Array.isArray(track.labels)) {
+        for (let i = 0; i < track.labels.length; i++) {
+          if (
+            track.labels[i].lang &&
+            player.language().indexOf(track.labels[i].lang.toLowerCase()) !== -1
```

Comment:
is `player.language()` always lower case? (I'm not sure)

Reply:
For the future, we do lowercase it in Video.js: https://github.com/videojs/video.js/blob/e612056bfa8e6e77ca6379e74ba14e9a75da32d6/src/js/player.js#L3912